projects/demo/src/pages/documentation/angular/examples/2-nested/template.html

```diff
@@ -0,0 +1,16 @@
+<!--
+  Maskito can be applied on this component, given the predicate
```

Comment:
Could you also rename cypress-file ?
https://github.com/Tinkoff/maskito/blob/main/projects/demo-integrations/cypress/tests/angular/di-approach.cy.ts

\+ add some new tests for this nested example with predicate